orchestration/templates/ingest-file.yaml

```diff
@@ -0,0 +1,153 @@
+apiVersion: argoproj.io/v1alpha1
+kind: WorkflowTemplate
+metadata:
+  name: ingest-file
+spec:
+  templates:
+    - name: main
+      inputs:
+        parameters:
+          - name: api-url
+          - name: dataset-id
+          - name: profile-id
+          - name: gcs-bucket
+          - name: gcs-file-path
+          - name: target-path
+          - name: timeout
+      dag:
+        tasks:
+          # 1) Make sure that something doesn't already exist at the target path
+          - name: check-existing-file
+            template: check-existing-file
+            arguments:
+              parameters:
+                - name: api-url
+                  value: '{{ "{{inputs.parameters.api-url}}" }}'
+                - name: dataset-id
+                  value: '{{ "{{inputs.parameters.dataset-id}}" }}'
+                - name: target-path
+                  value: '{{ "{{inputs.parameters.target-path}}" }}'
+          # 2) Make a post request to the Jade API to ingest the xml blob
+          - name: submit-file-ingest
+            template: submit-file-ingest
+            when: '{{ "tasks.check-existing-file.outputs.result}} == 200" }}'
```

Comment:
Is it going to read out the string "False" from Python or is it going to interpret it as a boolean (and does that even make a difference/matter here)? Since we are printing out from Python, I'm guessing we want to treat it like a string (asking because Python's boolean is a capitalized False)

Reply:
I think the script should print "false" or "true". Argo will see it as a string either way because its expression language is pretty simple (cry)

Reply:
Okay so we don't want to print the boolean built-ins in Python and we actually want to print the string "true" or "false" then, got it